publication-model/src/test/java/no/unit/nva/model/associatedartifacts/AssociatedArtifactListTest.java

```diff
@@ -0,0 +1,21 @@
+package no.unit.nva.model.associatedartifacts;
+
+import static no.unit.nva.model.testing.associatedartifacts.AssociatedArtifactsGenerator.randomOpenFile;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class AssociatedArtifactListTest {
+
+    @Test
+    void shouldRemoveNullAssociatedArtifactFromListWhenListContainsNotOnlyNullAssociatedArtifact() {
```

Comment:
What happens if you have two?

Reply:
Both will be removed. Or do we want to handle case:

When there is only NullAssociatedArtifacts in list, remove all keeping only one artifacts?

Reply:
I think it makes sense to say n+ > one